common/searchattribute/defs.go

```diff
@@ -35,6 +35,7 @@ import (
 const (
 	NamespaceID           = "NamespaceId"
 	WorkflowID            = "WorkflowId"
+	ScheduleID            = "ScheduleId"
```

Comment:
Can you move this to the bottom, and add some comments about how this is handled please?
Maybe together with TemporalNamespaceDivisision.